site/en/blog/richer-pwa-installation/index.md

```diff
@@ -0,0 +1,188 @@
+
+ 
+## Challenge with today's install surfaces {: #today }
+
+Today, if a user wants to install a PWA, an infobar and modal overlay appears with minimal
+information. If they do continue to install, the process is over too quickly without giving
+context to the user. This goes against their expectations of installing apps and can leave them
+somewhat confused with what has happened.  
+
+{% Img src="image/xizoeLGxYNf3VLUHc5BsIoiE1Af1/VwB1V3K61vQMs1htKJpY.png", 
+alt="An example of PWA install UI.", width="360", height="632" %}
+
+To enable and empower developers to provide installed experiences on--par with native experiences
+Chrome is introducing a new install surface, Richer Install, that will allow developers to add a
+description and screenshots to their manifest file and have it appear in a bottomsheet dialog
+within Chrome. 
+
+By doing this we are giving developers the opportunity to create a more enticing install process
+that better aligns to user expectations and better context that mimics their existing mental model
+of installed experiences. 
+
+{% Columns %}
+{% Column %}
+{% Img src="image/xizoeLGxYNf3VLUHc5BsIoiE1Af1/SpStAtUk8Zp5iwi9yqKP.jpg", 
+alt="Richer Install UI Expanded", width="342", height="722" %}
+{% Aside %}
+Richer Install UI Expanded
+{% endAside %}
+{% endColumn %}
+{% Column %}
+{% Img src="image/xizoeLGxYNf3VLUHc5BsIoiE1Af1/k7r4yKqrh6iOm2XyZHfw.jpg", 
+alt="Richer Install UI Collapsed", width="342", height="722" %}
+{% Aside %}
+Richer Install UI Collapsed
+{% endAside %}
+{% endColumn %}
+{% endColumns %}
+
+## Backwards compatibility {: #compatibility }
+
+Websites that do not conform to the above will continue to receive the existing prompts. This may
+change in the future depending on uptake of the developer community and users' reaction.  
+
+## Previewing the UI {: #previewing }
+
+This UI works in Chrome Dev & Chrome Canary on Android, with the flag enabled;
+
+`mobile-pwa-install-use-bottom-sheet` 
+
+This feature is enabled on [squoosh.app](https://squoosh.app) and can be previewed there. As an
+aside, [#screenshots](https://web.dev/add-manifest/#screenshots) and 
+[#description](https://web.dev/add-manifest/#description) fields are already documented; 
+you can give it a try behind the flag. 
+
+## Implementation {: #implementation }
+
+In order to enable Richer installs on your site you need to add at least one screenshot to your 
+manifest file. Descriptions are not required but are recommended. 
+
+Origins that are too long to fit the UI are truncated, this is also known as eliding and is used
+as a [security measure to protect users](https://chromium.googlesource.com/chromium/src/+/master/docs/security/url_display_guidelines/url_display_guidelines.md#eliding-urls). 
+
+Let's take a look at how we can trigger Richer Install UI when installing a PWA. In this example
+we are looking at squoosh.app implementation. Let's assume that you have your name and origin
+defined.
```

Comment:
Please rewrite without 'we' per the [handbook](https://web.dev/handbook/grammar/#instructions).

Reply:
please check this throughout.